sources/value-extractor-definition.asciidoc

```diff
@@ -106,8 +110,8 @@ Compatible implementations provide value extractors for the following types out
 * [tck-testable]#`java.util.Map`; both map keys and map values are to be supported;
 `keyedValue()` must be invoked by the map key extractor for each contained key, passing the string literal `<map key>` as node name;
 `keyedValue()` must be invoked by the map value extractor for each contained value, passing the string literal `<map value>` as node name#
-* [tck-testable]#`java.util.Optional`; `value()` must be invoked (passing the contained value or `null` if none is present), passing `null` as node name#
-* [tck-testable]#`java.util.OptionalInt`, `java.util.OptionalLong` and `java.util.OptionalDouble`; the extracted value types must be `java.lang.Integer`, `java.lang.Long` and `java.lang.Double`, respectively. `value()` must be invoked (passing the contained value or `null` if none is present), passing `null` as node name. The extractors must be marked with `@UnwrapByDefault`.#
+* [tck-testable]#`java.util.Optional`; `value()` must be invoked, passing `null` as node name and passing the contained object as value or `null` if none is present)#
```

Comment:
present) I can't find the starting parenthesis so I suppose we should remove it?